### ISSUE TITLE
Video

### ISSUE THREAD

**Issue by Jlomaka** [zhanziyang/vue-croppa#82]
Is it possible to work with video? It would be nice

**Comment by zhanziyang** [comment]
Maybe. I will see what can be done. Interesting idea 👍

**Comment by Jlomaka** [comment]
There is one small bug here, I'm downloading the video all right, I can play, flip and everything else, but only I press Cropp it turns it into a picture, it's possible to get some video, maybe there are extra parameters?

**Comment by zhanziyang** [comment]
"press" means click?

**Comment by Jlomaka** [comment]
When I execute this.myCroppa.generateDataUrl () I get png back, so I realized that I need another option to use, I need to use something else?

**Comment by zhanziyang** [comment]
Yes, generateDataUrl won't give you a video. You need to use the technique introduced in these links:
https://developers.google.com/web/updates/2016/10/capture-stream
https://developers.google.com/web/updates/2016/01/mediarecorder

**Comment by teilzeitgeist** [comment]
Hey!

I tried to use the option `:video-enabled=true` and I get a video object. Cool.

My problem is croppa do not get the first frame and draw it on the canvas. So right now I only see a transparent image and unfortunately there is no error.

Here a simple CodePen to reproduce the problem:
https://codepen.io/psyphant/pen/vYOQrZO

**Comment by zhanziyang** [comment]
On vue-croppa 2.x: Set `videoEnabled: true` to accept and crop browser-playable video files (double-click canvas to play/pause).

Closing as answered. Please feel free to open a new issue if this doesn't cover your use case.
